### ISSUE TITLE
[help wanted] do i have to install tsshd on my jump server

### ISSUE THREAD

**Issue by isvicy** [trzsz/tsshd#5]
i have to use a jump server to access the dev server i use. i want to know, if i have to install tsshd and tssh on the jump server to use the udp mode as mosh.

**Comment by lonnywong** [comment]
Yes, you have to install tsshd and tssh on the jump server. Assuming you installed them in `/home/you/bin/tssh` and `/home/you/bin/tsshd`.

UDP mode is hard to work with ProxyJump and DynamicForward. You have to login to the jump server first, then use the `Automated Interaction` feature to login to target server.

```
Host jp-dev
    #!! UdpMode yes
    #!! TsshdPath /home/you/bin/tsshd
    HostName t-nix
    LocalForward 1080 127.0.0.1:1080
    #!! ExpectCount 1
    #!! ExpectPattern1 $
    #!! ExpectSendText1 /home/you/bin/tssh --udp --tsshd-path /home/you/bin/tsshd -p 3200 -D 1080 user@hostname
```

**Comment by lonnywong** [comment]
> and the tsshd and tssh binary is built and install to a dir in my home directory, it could not be directly found in the login shell, so i need to specify the binary location in ssh config, how?

Specify the binary location like:
```
Host jump_server
    #!! UdpMode yes
    #!! TsshdPath ~/go/bin/tsshd
```
